emsdk/Makefile

```diff
@@ -4,6 +4,7 @@ all: emsdk/emsdk
 # run out of memory
 
 emsdk/emsdk:
+	if [ -d emsdk ]; then rm -rf emsdk; fi
 	git clone https://github.com/juj/emsdk.git
```

Comment:
Maybe,
```
git -C emsdk pull || git clone https://github.com/juj/emsdk.git
```
(taken from https://stackoverflow.com/a/31186538/1791279) 

to pull if the repo exists and clone if it doesn't?

Otherwise this would be a 0.5MB download each time one runs make to re-download it..

Reply:
That's not a bad idea.  I'll update this PR.